appveyor.yml

```diff
@@ -0,0 +1,29 @@
+platform:
+  - x64
+environment:
+  RUST_INSTALL_DIR: C:\Rust
+  matrix:
+    - RUST_INSTALL_TRIPLE: x86_64-pc-windows-msvc
+      VCVARS: "C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\bin\\amd64\\vcvars64.bat"
+      RUST_VERSION: 1.34.2
+
+install:
+  - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-${env:RUST_VERSION}-${env:RUST_INSTALL_TRIPLE}.exe"
+  - cmd: rust-%RUST_VERSION%-%RUST_INSTALL_TRIPLE%.exe /VERYSILENT /NORESTART /DIR="%RUST_INSTALL_DIR%"
+  # Use clean path to work around cyghead base mismatch issue
+  - cmd: SET PATH=C:\Windows\System32;C:\Windows;C:\Windows\System32\Wbem;C:\msys64\usr\bin
+  - cmd: CALL "%VCVARS%"
+  - cmd: SET PATH=%PATH%;%RUST_INSTALL_DIR%\bin
+  - rustc --version
+  - cargo --version
+  - make --version
+  - cp --version
+
+build: false
+test_script:
+  - cmd: SET RUST_BACKTRACE=full
+  # The plan we are using for appveyor now cannot finish unit tests
+  # and integration tests in one run, so we have to separate them to
+  # run unit in branches, and integration tests in PRs
+  - if NOT defined APPVEYOR_PULL_REQUEST_NUMBER make test
+  - if defined APPVEYOR_PULL_REQUEST_NUMBER make integration-windows
```

Comment:
The condition is inversed. We only run unit tests for PR, and integration for branch.